src/main/java/com/minecolonies/coremod/items/ItemSupplyChestDeployer.java

```diff
@@ -80,6 +79,17 @@
     private static final int OFFSET_Y = -2;
 
     /**
+     * Our guide Book.
+     */
+    private static final ResourceLocation GUIDE_BOOK = new ResourceLocation(Constants.MOD_ID + ":book/minecolonies.xml");
+
+    /**
+     * Our guide Book.
+     */
+    @GameRegistry.ItemStackHolder(value = "gbook:guidebook", nbt = "{Book:\"minecolonies:book/minecolonies.xml\"}")
+    public static ItemStack guideBook;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static guideBook" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1444)